### ISSUE TITLE
How to pass custom dataset?

### ISSUE THREAD

**Issue by hinash88** [znxlwm/pytorch-MNIST-CelebA-cGAN-cDCGAN#3]
Can Anybody please help how to pass custom datasets instead of MNIST?

`train_loader = torch.utils.data.DataLoader(
    datasets.MNIST('data', train=True, download=False, transform=transform),
    batch_size=batch_size, shuffle=True)`
